packages/regl-worldview/src/Worldview.js

```diff
@@ -201,8 +220,10 @@ export class WorldviewBase extends React.Component<BaseProps, State> {
       return;
     }
 
-    // rendering the hitmap on mouse move is expensive, so disable it by default
-    if (mouseEventName === "onMouseMove" && !this.props.hitmapOnMouseMove) {
+    // Rendering the hitmap is expensive, so we disable it for some events
+    // Rendering the hitmap on mouse move is disabled by default.
+    const { hitmapOnMouseMove, disableHitmapForEvents = ["onMouseMove"] } = this.props;
```

Comment:
```suggestion
    const { disableHitmapForEvents = hitmapOnMouseMove ? [] : ["onMouseMove"] } = this.props;
```

or so? I find the clause below a bit hard to reason about, so it might be better to show exactly how `hitmapOnMouseMove` gets migrated to `disableHitmapForEvents`.

Reply:
I'm not sure about that change. 
My understanding is that `hitmapOnMouseMove` should override only what `disableHitmapForEvents` does for `onMouseMove`. In other words, if `disableHitmapForEvents` specifies that both `onMouseDown` and `onMouseMove` events should be ignored, using `hitmapOnMouseMove` leads to Worldview ignoring only `onMouseDown` for hitmap rendering.
Based on your suggested change, by using `hitmapOnMouseMove` we will be triggering the hitmap on a `onMouseDown` event too since we're clearing `disableHitmapForEvents`. 
Does this makes sense?

Reply:
We're deprecating `hitmapOnMouseMove` though right? Since it will be completely superseded by `disableHitmapForEvents`? So I think we can assume that if a user has `disableHitmapForEvents` set, that they won't have `hitmapOnMouseMove` set. In fact, maybe we should throw an error if both are set, to avoid confusion.